ios/FluentUI/Card Nudge/MSFCardNudge.swift

```diff
@@ -18,6 +18,7 @@ import UIKit
                       title: String) {
         let cardNudge = CardNudge(style: style, title: title)
         state = cardNudge.state
+        tokenResolver = cardNudge.tokenResolver
```

Comment:
I think it'd be nice to see this in a composite in UIKit too.
It'll probably help us validate the token resolver in that scenario.
Maybe **DrawerController** is a good one?

Reply:
Sure! I was originally going to save that for a different PR, but I agree that having one example of each will be helpful. Let me finalize that now.